server/tso/allocator_manager.go

```diff
@@ -592,11 +589,21 @@ func (am *AllocatorManager) getAllocatorGroups(filters ...AllocatorGroupFilter)
 	return allocatorGroups
 }
 
+func (am *AllocatorManager) getAllocatorGroup(dcLocation string) (*allocatorGroup, bool) {
+	am.mu.RLock()
+	defer am.mu.RUnlock()
+	if len(dcLocation) == 0 {
```

Comment:
Because this part of logic is only available and used in the `HandleTSORequest` method, so if you make it an independent function like `getAllocatorGroup` now, maybe we should add some comments or log to tell the user why we do this?

Reply:
good catch, updated.